core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java

```diff
@@ -178,6 +181,16 @@ private void registerEth2Routes(
               new ImportKeystoresHandler(
                   objectMapper, config.getKeyConfigPath(), slashingProtection, blsSignerProvider),
               false));
+
+      routerFactory.addHandlerByOperationId(
+          KEYMANAGER_DELETE.name(),
+          new BlockingHandlerDecorator(
+              new DeleteKeystoresHandler(
+                  objectMapper,
+                  new KeystoreFileManager(config.getKeyConfigPath()),
+                  slashingProtection,
+                  blsSignerProvider),
+              false));
```

Comment:
Does this need the errorHandler adding as failure handler? I think same applies to the list and import operations too.

Reply:
Maybe a question for @jframe

Reply:
Yes we should add that. That will add some additional logging if the request fails.

Reply:
done